[13.0] [DEL] company_country: leave only module shell

NL66278 opened the pull request:
The module is no longer needed on Odoo 13.0 or above,
but is causing a crash of runboat.

NL66278 commented:
@moylop260 @luisg123v Do you agree this module can be removed? It is causing errors in runboat, thereby putting all 13.0 PR's in failure state.

luisg123v commented:
Hi @NL66278,

This module can't be removed without affecting already-running instances with this module installed. For more info, see discussion at https://github.com/OCA/server-tools/pull/1872, specially https://github.com/OCA/server-tools/pull/1872#issuecomment-670965398.

Regards,

CC @astirpe @legalsylvain

NL66278 committed:
[DEL] company_country: leave only shell of module.

Module is no longer needed as problem solved in Odoo >- 13.0. However module causes
problems with runboat, leading to failures in all 13.0 PR's. Simply deleting the
module might cause problems with existing installations. Therefore a shell is kept.

luisg123v commented:
/ocabot merge major
